homeassistant/components/downloader/services.yaml

```
@@ -1,5 +1,6 @@
 download_file:
-  description: Downloads a file to the download location
+  name: Download file
+  description: Downloads a file to the download location.
```

Shouldn't we use imperative consistently in the description?
```suggestion
  description: Download a file to the download location.
```